authors/apps/articles/views.py

```diff
@@ -16,8 +18,55 @@
 class ArticlesApiView (generics.ListCreateAPIView):
     permission_classes = (permissions.IsAuthenticatedOrReadOnly,)
     serializer_class = serializers.ArticleSerializer
-    queryset = Article.objects.all()
     pagination_class = ArticlesLimitOffsetPagination
+    queryset = Article.objects.all()
+    search_fields = (
+        'title',
+        'body',
+        'description',
+        'tagList',
+        'author__username',
+        'favorited_articles'
+    )
+
+    def get_queryset(self):
```

Comment:
Thanks for the good work @BagzieGracious, however, the function `get_queryset` has a Cognitive Complexity of 6 (exceeds 5 allowed). You can consider refactoring.